### PR TITLE
Edit score properties in WebView on musescore.com upload

### PR DESCRIPTION
**Pull request by dmitrio95**
This PR implements changes to a process of uploading scores to musescore.com according to which score properties are edited in a web view (or a web browser in case QWebEngine is not available). This will allow to resolve an issue with synchronizing possible score settings between MuseScore editor and musescore.com service.

### CODE REVIEW

File: mscore/uploadscoredialog.ui [hunk 1]
```diff
@@ -65,241 +84,94 @@
...
      </item>
      <item>
-      <layout class="QGridLayout" name="gridLayout_4">
-       <item row="0" column="0">
+      <layout class="QHBoxLayout" name="horizontalLayout_7">
+       <item>
         <widget class="QCheckBox" name="updateExistingCb">
          <property name="text">
-          <string>Update existing score</string>
+          <string notr="true">Update existing score</string>
```
Comment (Jojo-Schmitz) [inline mscore/uploadscoredialog.ui]: why not translate?

Comment (dmitrio95) [reply mscore/uploadscoredialog.ui]: This must be a leftover from attempts to merge it with the "[Link]" label, thanks for catching!

File: mscore/uploadscoredialog.ui [hunk 2]
```diff
@@ -18,13 +18,32 @@
...
+        </size>
+       </property>
+       <property name="maximumSize">
+        <size>
+         <width>48</width>
+         <height>48</height>
+        </size>
+       </property>
+       <property name="text">
+        <string notr="true">User</string>
```
Comment (Jojo-Schmitz) [inline mscore/uploadscoredialog.ui]: and this `notr`?

File: mscore/uploadscoredialog.ui [hunk 3]
```diff
@@ -65,241 +84,94 @@
...
-          <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
-         </property>
-         <property name="tabChangesFocus">
-          <bool>true</bool>
-         </property>
-        </widget>
-       </item>
-      </layout>
+       <property name="text">
+        <string notr="true">Icon</string>
```
Comment (Jojo-Schmitz) [inline mscore/uploadscoredialog.ui]: and this `notr`?

File: mscore/uploadscoredialog.ui [hunk 4]
```diff
@@ -65,241 +84,94 @@
...
-       </item>
-      </layout>
+       <property name="maximumSize">
+        <size>
+         <width>16777215</width>
+         <height>48</height>
+        </size>
+       </property>
+       <property name="text">
+        <string notr="true">Title</string>
```
Comment (Jojo-Schmitz) [inline mscore/uploadscoredialog.ui]: and this `notr`?

File: mscore/uploadscoredialog.ui (hunk 2 above)
Comment (dmitrio95) [reply mscore/uploadscoredialog.ui]: These `notr`s are intentional, these strings are just placeholders which are replaced to actual strings or icons in C++ code.

Comment (Jojo-Schmitz) [reply mscore/uploadscoredialog.ui]: OK. On other such cases an `extracomment` is there to explain this.

**Pull request merged**